src/aiida_quantumespresso/calculations/pw.py

```diff
@@ -146,6 +146,9 @@ def define(cls, spec):
             message='Then ionic minimization cycle converged but the thresholds are exceeded in the final SCF.')
         spec.exit_code(502, 'ERROR_IONIC_CYCLE_EXCEEDED_NSTEP',
             message='The ionic minimization cycle did not converge after the maximum number of steps.')
+        spec.exit_code(503, 'ERROR_SCALE_H',
+            message=('The cell relaxation caused a significant volume contraction '
+                     'and there is not enoguh space allocated for radial FFT.'))
```

Comment:
Open it again. Actually, this can also be a __significant expansion__. Shall we change the name to e.g. __VOLUME_SCALING__ or similar?

Reply:
If it can also relate to expansion, then I agree that is a better name. Currently you used `_CONTRACTION` still though? Did you still need to adapt this?

Reply:
In case yes, but from the reply of Paolo Giannozzi, this error seems to appear indeed only for large contraction, and not for expansions.